docs/samples/Create Asset.md

```diff
@@ -0,0 +1,295 @@
+              },
+              "edc:asset": {
+                "@type": "edc:asset",
+                "@id": "some-asset-id",
+                "properties": {
+                  "name": "some-asset-name",
+                  "description": "some description",
+                  "edc:version": "0.2.1",
+                  "contenttype": "application/json"
+                }
+              },
+              "edc:dataAddress": {
+                "@type": "edc:dataAddress",
+                "type": "test-type",
+                "keyName": "test-key-name"
+              }
+            }' \
+    -s -o /dev/null -w 'Response Code: %{http_code}\n'
+```
+
+## 6. Private Property Asset
+
+A new addition are the private properties.
+Private properties will not be sent through the dataplane and are only accessible via the management API.
+This enables the storage of additional information pertaining the asset, that is not relevant for the consumer, but is nonetheless useful for the provider.
+Private properties are stores inside the `privateProperties` field.
+
+```json
+{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
+    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "edc:asset": {
+    "@type": "edc:asset",
+    "@id": "some-asset-id",
+    "properties": {
+      "name": "some-asset-name",
+      "description": "some description",
+      "edc:version": "0.2.1",
+      "contenttype": "application/json"
+    },
+    "edc:privateProperties": {
+      "test-prop": "test-val"
+    }
+  },
+  "edc:dataAddress": {
+    "@type": "edc:dataAddress",
+    "type": "test-type",
+    "keyName": "test-key-name"
+  }
+}
+```
+
+```bash
+curl -X POST "${CON_DATAMGMT_URL}/management/v2/assets" \
+    --header 'X-Api-Key: password' \
+    --header 'Content-Type: application/json' \
+    --data '{
+              "@context": {
+                "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
+                "edc": "https://w3id.org/edc/v0.0.1/ns/"
+              },
+              "edc:asset": {
+                "@type": "edc:asset",
+                "@id": "some-asset-id",
+                "properties": {
+                  "name": "some-asset-name",
+                  "description": "some description",
+                  "edc:version": "0.2.1",
+                  "contenttype": "application/json"
+                },
+                "edc:privateProperties": {
+                  "test-prop": "test-val"
+                }
+              },
+              "edc:dataAddress": {
+                "@type": "edc:dataAddress",
+                "type": "test-type",
+                "keyName": "test-key-name"
+              }
+            }' \
+    -s -o /dev/null -w 'Response Code: %{http_code}\n'
+```
+
+## 7. Complex Property Asset
+
+Besides primitive datatypes, complex JSON objects can also be stored inside the properties.
+An example is the `payload` field and its contents.
+Here the `payload` contains the name and age of a person.
```

Comment:
Can it be anything? Because then I wouldn't use the term `payload`. This sounds like, that I can store the to be transmitted data within this field.